epictrack-api/src/api/actions/set_work_state.py

```diff
@@ -25,8 +26,10 @@ def run(self, source_event, params) -> None:
             }
             change_phase_end_event.run(source_event, change_phase_end_event_param)
         is_active = True
+        ended_at = None
         if work_state in [state.value for state in EndingWorkStateEnum]:
             is_active = False
+            ended_at = datetime.now()
         db.session.query(Work).filter(Work.id == source_event.work_id).update(
-            {Work.work_state: work_state, Work.is_active: is_active}
+            {Work.work_state: work_state, Work.is_active: is_active, Work.ended_at: ended_at}
```

Comment:
can't you use "work_decision_date" column for this purpose? I can see that this column is not used anywhere. But the plan is to set it when the decision is made on that work.

Reply:
yea i see its not used anywhere. what decision is it reffering to?

Reply:
Whenever the work state changes to one of the ending_states

Reply:
oh ok. ill refactor to use that